utils/utils.go

```diff
@@ -123,3 +124,15 @@ func StripFromString(input, substring string) string {
 
 	return strippedString
 }
+
+// Converts a '0x'-prefixed hex string or cb58-encoded string to an ID.
```

Comment:
Can you add a comment here that input length validation is handled by the `ids` package?

Reply:
Looks like this comment was resolved, but I don't see a corresponding commit.

Reply:
Sorry, I pushed but I think I typed my yubikey pin wrong 😓